pkg/services/control/ir/service.proto

```diff
@@ -41,3 +50,89 @@ message HealthCheckResponse {
     // Body signature.
     Signature signature = 2;
 }
+
+// NotaryList request.
+message NotaryListRequest {
+    // Request body structure.
+    message Body {
+    }
+
+    // Body of request message.
+    Body body = 1;
+
+    // Body signature.
+    Signature signature = 2;
+}
+
+// NotaryList response.
+message NotaryListResponse {
+    // Request body structure.
+    message Body {
+        // Transaction of the notary request.
+        repeated TransactionInfo transactions = 1;
+    }
+
+    // Body of request message.
+    Body body = 1;
+
+    // Body signature.
+    Signature signature = 2;
+}
+
+// NotaryRequest request.
+message NotaryRequestRequest {
+    // Request body structure.
+    message Body {
+        string method = 1;
```

Comment:
maybe it would be better to have a more generic `contractHash, method, args` or maybe not @roman-khimov, @cthulhu-rider

Reply:
Let's keep it simple for now, can be extended in future.